README.md

```diff
@@ -1,116 +1,80 @@
+
+Regularly tested means we try to run our full test suite against that set of devices prior to each Netmiko release.
+
+Limited testing means the config and show operation system tests passed against a test on that platform at one point in time so we are reasonably comfortable the driver should generally work.
+
+Experimental means that we reviewed the PR and the driver seems reasonable, but we don't have good data on whether the driver fully passes the unit tests or how reliably it works.
+
+Click [here](PLATFORMS.md) for a list of all devices.
+
+
+## Installation:
+
+To install netmiko, simply us pip:
+
+```
+$ pip install netmiko
+```
+
+Netmiko has the following requirements (which pip will install for you)
 - Paramiko >= 2.4.2
 - scp >= 0.10.0
 - pyyaml
 - pyserial
 - textfsm
 
-#### Supports:
-
-###### Regularly tested
-
-- Arista vEOS
-- Cisco ASA
-- Cisco IOS
-- Cisco IOS-XE
-- Cisco IOS-XR
-- Cisco NX-OS
-- Cisco SG300
-- HP ProCurve
-- Juniper Junos
-- Linux
-
-###### Limited testing
-
-- Alcatel AOS6/AOS8
-- Apresia Systems AEOS
-- Calix B6
-- Cisco AireOS (Wireless LAN Controllers)
-- CloudGenix ION
-- Dell OS9 (Force10)
-- Dell OS10
-- Dell PowerConnect
-- Extreme ERS (Avaya)
-- Extreme VSP (Avaya)
-- Extreme VDX (Brocade)
-- Extreme MLX/NetIron (Brocade/Foundry)
-- HP Comware7
-- Huawei
-- IP Infusion OcNOS
-- Mellanox
-- MikroTik RouterOS
-- MikroTik SwitchOS
-- NetApp cDOT
-- OneAccess
-- Palo Alto PAN-OS
-- Pluribus
-- Ruckus ICX/FastIron
-- Ubiquiti EdgeSwitch
-- Vyatta VyOS
-
-###### Experimental
-
-- A10
-- Accedian
-- Aruba
-- Ciena SAOS
-- Citrix Netscaler
-- Cisco Telepresence
-- Check Point GAiA
-- Coriant
-- Dell OS6
-- Dell EMC Isilon
-- Eltex
-- Enterasys
-- Extreme EXOS
-- Extreme Wing
-- Extreme SLX (Brocade)
-- F5 TMSH
-- F5 Linux
-- Fortinet
-- Mikrotik
-- MRV Communications OptiSwitch
-- Nokia/Alcatel SR-OS
-- QuantaMesh
-- Rad ETX
-- Versa Networks FlexVNF
-
-## Tutorials:
-
-##### Standard Tutorial:
-
-https://pynet.twb-tech.com/blog/automation/netmiko.html
-
-##### Secure Copy Tutorial:
-
-https://pynet.twb-tech.com/blog/automation/netmiko-scp.html
-
-##### SSH Proxy:
-
-https://pynet.twb-tech.com/blog/automation/netmiko-proxy.html
-
-##### Common Issues:
-
-https://github.com/ktbyers/netmiko/blob/develop/COMMON_ISSUES.md
-
-##### Documentation (Stable)
-
-http://netmiko.readthedocs.io/en/stable/index.html
-
-## Examples:
+
+## Documentation
+
+<a href="https://ktbyers.github.io/netmiko/docs/netmiko/index.html" title="Docs">API Documentation</a>
+
+Below are some of the particularly handy Classes/functions for easy reference:
+- [Base Connection Object](https://ktbyers.github.io/netmiko/docs/netmiko/base_connection.html)
+- [SSH Autodetect](https://ktbyers.github.io/netmiko/docs/netmiko/index.html#netmiko.SSHDetect)
+- [SSH Dispatcher](https://ktbyers.github.io/netmiko/docs/netmiko/index.html#netmiko.ssh_dispatcher)
+- [Redispatch](https://ktbyers.github.io/netmiko/docs/netmiko/index.html#netmiko.redispatch)
```

Comment:
I would probably add the SCP file_transfer and some of that in here (probably as the second most common one after BaseConnection.

Reply:
It would also be good to link to these example/reference code here for this set of common use cases (in some way). Not sure if this is linked to on the readme in any way.

Reply:
Meant to add this link: https://github.com/ktbyers/netmiko/tree/develop/examples/use_cases